packages/valory/skills/abstract_round_abci/base.py

```diff
@@ -878,6 +883,11 @@ def participant_to_votes(self) -> Mapping:
         """Check whether keeper is set."""
         return cast(Dict, self.db.get_strict("participant_to_votes"))
 
+    @property
+    def safe_contract_address(self) -> str:
```

Comment:
Please check it is removed from all synchronised data children

Reply:
See the comment above.

Reply:
this is not covered by a unit test 
```
+    @property
+    def safe_contract_address(self) -> str:
+        """Get the safe contract address."""
+        return cast(str, self.db.get_strict("safe_contract_address"))   # <----
```